servers/dsub/jobs/controllers/aggregation_controller.py

```diff
@@ -24,47 +16,98 @@ def get_job_aggregations(timeFrame, projectId=None):
     Returns:
         AggregationResponse: Response contains aggregation of jobs
     """
-    # temporary fake data for testing front-end.
-    statusCount1 = StatusCount(status='Succeeded', count=10)
-
-    statusCount2 = StatusCount(status='Failed', count=2)
-
-    summaryCounts = StatusCounts(counts=[statusCount1, statusCount2])
-
-    entry1 = AggregationEntry(
-        label='labelValue1',
-        status_counts=StatusCounts(counts=[
-            StatusCount(status='Succeeded', count=2),
-            StatusCount(status='Failed', count=1)
-        ]))
-
-    entry2 = AggregationEntry(
-        label='labelValue2',
-        status_counts=StatusCounts(counts=[
-            StatusCount(status='Succeeded', count=4),
-            StatusCount(status='Failed', count=6)
-        ]))
-
-    userEntry1 = AggregationEntry(
-        label='circleci',
-        status_counts=StatusCounts(counts=[
-            StatusCount(status='Succeeded', count=3),
-            StatusCount(status='Failed', count=7)
-        ]))
-
-    userEntry2 = AggregationEntry(
-        label='otherUser',
-        status_counts=StatusCounts(counts=[
-            StatusCount(status='Succeeded', count=7),
-            StatusCount(status='Failed', count=4)
-        ]))
-
-    ownerAggregation = Aggregation(
-        name='AnotherLabel', key='job-id', entries=[entry1, entry2])
-
-    projectAggregation = Aggregation(
-        name='User', key='userId', entries=[userEntry1, userEntry2])
+    window_min = time_convert.time_frame_to_start_time(timeFrame)
+    provider = providers.get_provider(generator.provider_type(), projectId,
+                                      generator.auth_token())
+
+    jobs = generator.generate_jobs_by_window(provider, projectId, window_min)
+
+    total_summary = {}
+    user_summary = {}
+    job_name_summary = {}
+
+    for job in jobs:
+        _count_total_summary(job, total_summary)
+        _count_user_summary(job, user_summary)
+        _count_job_name_summary(job, job_name_summary)
+
+    return _get_stub_data(
+        _to_summary_counts(total_summary),
+        _to_user_aggregation_entry(user_summary),
+        _to_job_name_aggregation_entry(job_name_summary))
+
+
+def _count_total_summary(job, total_summary):
+    if job.status not in total_summary:
+        total_summary[job.status] = 0
+    total_summary[job.status] += 1
+
+
+def _to_summary_counts(summary_counts):
```

Comment:
This is a trivial function I would just inline it above and shorten the vars:
`[StatusCount(s, c) for s, c in total_summary.items()]`